api/vulnerability_exceptions.go

```diff
@@ -0,0 +1,453 @@
+// 'Critical', 'High', 'Medium', 'Low' or 'Info'
+type vulnerabilityExceptionSeverity int
+
+type VulnerabilityExceptionSeverities []vulnerabilityExceptionSeverity
+
+//nolint
+func (sevs VulnerabilityExceptionSeverities) toInt() []int {
+	var res []int
+	for _, i := range sevs {
+		res = append(res, int(i))
+	}
+	return res
+}
+
+func (sevs VulnerabilityExceptionSeverities) ToStringSlice() []string {
+	var res []string
+	for _, i := range sevs {
+		switch i {
+		case VulnerabilityExceptionSeverityCritical:
+			res = append(res, "Critical")
+		case VulnerabilityExceptionSeverityHigh:
+			res = append(res, "High")
+		case VulnerabilityExceptionSeverityMedium:
+			res = append(res, "Medium")
+		case VulnerabilityExceptionSeverityLow:
+			res = append(res, "Low")
+		case VulnerabilityExceptionSeverityInfo:
+			res = append(res, "Info")
+		default:
+			continue
+		}
+	}
+	return res
+}
+
+func NewVulnerabilityExceptionSeverities(sevSlice []string) VulnerabilityExceptionSeverities {
+	var res VulnerabilityExceptionSeverities
+	for _, i := range sevSlice {
+		sev := convertVulnerabilityExceptionSeverity(i)
+		if sev != VulnerabilityExceptionSeverityUnknown {
+			res = append(res, sev)
+		}
+	}
+	return res
+}
+
+func NewVulnerabilityExceptionSeveritiesFromIntSlice(sevSlice []int) VulnerabilityExceptionSeverities {
+	var res VulnerabilityExceptionSeverities
+	for _, i := range sevSlice {
+		sev := convertVulnerabilityExceptionSeverityInt(i)
+		if sev != VulnerabilityExceptionSeverityUnknown {
+			res = append(res, sev)
+		}
+	}
+	return res
+}
+
+func convertVulnerabilityExceptionSeverity(sev string) vulnerabilityExceptionSeverity {
+	switch strings.ToLower(sev) {
+	case "critical":
+		return VulnerabilityExceptionSeverityCritical
+	case "high":
+		return VulnerabilityExceptionSeverityHigh
+	case "medium":
+		return VulnerabilityExceptionSeverityMedium
+	case "low":
+		return VulnerabilityExceptionSeverityLow
+	case "info":
+		return VulnerabilityExceptionSeverityInfo
+	default:
+		return VulnerabilityExceptionSeverityUnknown
+	}
+}
+
+func convertVulnerabilityExceptionSeverityInt(sev int) vulnerabilityExceptionSeverity {
+	switch sev {
+	case 1:
+		return VulnerabilityExceptionSeverityCritical
+	case 2:
+		return VulnerabilityExceptionSeverityHigh
+	case 3:
+		return VulnerabilityExceptionSeverityMedium
+	case 4:
+		return VulnerabilityExceptionSeverityLow
+	case 5:
+		return VulnerabilityExceptionSeverityInfo
+	default:
+		return VulnerabilityExceptionSeverityUnknown
+	}
+}
+
+const (
+	VulnerabilityExceptionSeverityCritical vulnerabilityExceptionSeverity = 1
+	VulnerabilityExceptionSeverityHigh     vulnerabilityExceptionSeverity = 2
+	VulnerabilityExceptionSeverityMedium   vulnerabilityExceptionSeverity = 3
+	VulnerabilityExceptionSeverityLow      vulnerabilityExceptionSeverity = 4
+	VulnerabilityExceptionSeverityInfo     vulnerabilityExceptionSeverity = 5
+	VulnerabilityExceptionSeverityUnknown  vulnerabilityExceptionSeverity = 0
```

Comment:
why do we need numbers?

Reply:
We do not need this, the severities here are different that other schemas